scipy/sparse/csgraph/_disjoint_set.pyx

```diff
@@ -0,0 +1,130 @@
+"""
+Disjoint set data structure
+"""
+
+import numpy as np
+cimport numpy as np
+cimport cython
+
+
+cdef class DisjointSet:
+    """Disjoint set data structure [1]_.
```

Comment:
This seems to render weird in the [docs](https://20588-1460385-gh.circle-artifacts.com/0/html-scipyorg/sparse.csgraph.html?highlight=csgraph#module-scipy.sparse.csgraph) maybe move to a 2nd line?

Reply:
I have moved the reference into the notes section. Hopefully that will fix it.